locksmith/src/operations/lockOperations.ts

```diff
@@ -115,7 +115,8 @@ export async function updateLockMigrationsLog(
 
 export async function isSoldOut(
   address: string,
-  chain: number
+  chain: number,
+  keysNeeded = 10
```

Comment:
@searchableguy default value for `keysNeeded` shout not be equal to zero?

Reply:
The default is high to provide a reasonable estimation of *are there enough keys for me to buy?*